lib/typescript.js

```diff
@@ -55,6 +55,18 @@ function convertType(swaggerType, swagger) {
             var property = convertType(propertyType);
             property.name = propertyName;
             typespec.properties.push(property);
+            // see if property is required
```

Comment:
This change and the one in `type.mustache` seems to be unrelated to the other change with the response, is my understanding correct?